.github/workflows/students-workflow.yml

```diff
@@ -0,0 +1,19 @@
+name: Workflow for students # TODO: find a better name
```

Comment:
What do you think? Probably we should even move this workflow setup to a separate branch with its own PR, and for now just merge basic solution structure.

I can move this commit with a workflow to another branch